wordpress-mobile/WordPress-iOS
# Menus: remote menus service and tests

Adds the Menus remote service and unit tests. The Menus API is only available over the REST API, so no XML-RPC remote is needed here.

- `RemoteMenu` is a site menu itself.
- `RemoteMenuLocation` is a location/area in which a menu can appear on a site/theme.
- `RemoteMenuItem` are the individual items/link for a menu.

To test:
- Run the tests under `MenusServiceRemoteTests` and check for all passing.

Needs review: @diegoreymendez would please review? 💃💃💃
